packages/tasit-action/src/contract/Contract.js

```diff
@@ -163,9 +168,14 @@ export class Contract extends Subscription {
       if (!Utils.isEthersJsSigner(this.#ethersContract.signer))
         throw new Error(`Cannot write data to a Contract without a wallet`);
 
-      const tx = this.#ethersContract[f.name].apply(null, args);
+      const { signer, address: to } = this.#ethersContract;
+      const data = this.#ethersContract.interface.functions[f.name].encode(
+        args
+      );
+
+      const rawTx = { to, data };
```

Comment:
I think it might be easier to read if it said "to: address" here instead of renaming address to to a few lines up. Because I wondered a few lines up why it was being called to, but once I got here it made sense.

Reply:
Done.

Reply:
Thanks! Resolving